config/application.rb

```diff
@@ -67,6 +67,7 @@ class Application < Rails::Application
         )
       config.registry_public_key = config.registry_private_key.public_key
       config.access_policies = '/etc/rmt/access_policies.yml'
+      config.regisry_cache_dir = '/run/rmt/cache/registry'
```

Comment:
s/regisry/registry/

Reply:
This seems to be the only change in `config/application.rb`. Should it be moved along with the other configs (in `production.rb` I think?)

https://github.com/SUSE/rmt/pull/1141/files#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eL23

Reply:
Yes, for this PR is OK
@digitaltom suggested to move `Registry` engine config info to the config inside the engine, which makes sense to me but I do not think it should be part of the PRs of registry _cache_

thoughts ?

Reply:
The suggested move is good, and I agree that can come in a separate PR.

But, this one line is the only change now on `config/application.rb` - so let's be consistent with the rest of the PR and move it in with the rest of the changes... then `config/application.rb` doesn't get touched.